pandas/tests/test_internals.py

```diff
@@ -286,6 +288,36 @@ def test_repr(self):
         pass
 
 
+class TestDatetimeBlock(tm.TestCase):
+    _multiprocess_can_split_ = True
+
+    def test_try_coerce_arg(self):
+        block = create_block('datetime', [0])
+
+        # coerce None
+        none_coerced = block._try_coerce_args(block.values, None)[1]
+        self.assertTrue(pd.Timestamp(none_coerced) is pd.NaT)
+
+        # coerce np.datetime64
```

Comment:
Many people (including me) are against the use of for-loops in unit-tests. We believe tests are different from production code, and being stupid clear and simple is more important than having a bit of duplicated code. 

Anyway, I will do as you wish, I just want to point out that was a conscious choice...